### DISCUSSION TITLE
Disable Copilot on specific files.

### DISCUSSION THREAD

**Discussion by Victorsitou** [community/community#6644] [Copilot Conversations] [answered]
Hi, this is my first day using Copilot and I think it is a wonderful tool for developers. 

I would like to suggest the possibility to disable Copilot only on specific files. I bet the team is working on this, but if not, then it would be a nice feature to implement.

**Comment by hsenag** [comment, marked as answer]
Hi @Victorsitou!

At present you can [disable it for specific languages](https://github.com/github/copilot-preview/blob/main/docs/configuringvisualstudiocode.md#settings-for-the-extension) but not for individual files only.

**Reply by all-iver** [reply]
Just commenting to say I agree that this is an important feature to add, maybe an ignore file or something like that.  That way users could exclude files with sensitive data or 3rd party licensing.

**Reply by BarnabasSzabolcs** [reply]
Yes, I don't want copilot to scrape up my passwords from the env files.

**Comment by whoan** [comment]
This is an up-to-date link to [disable Copilot for specific languages](https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment#configuring-language-settings-for-github-copilot).
